lib/Geo/IP.pm

```diff
@@ -5405,10 +5405,33 @@ sub org_by_name {
     return $gi->org_by_addr( $gi->get_ip_address($host) );
 }
 
+sub org_by_name_v6 {
+    my ( $gi, $host ) = @_;
+    return $gi->org_by_addr_v6( $gi->get_ip_address($host) );
+}
+
 #this function returns isp or org of the domain name
 sub org_by_addr {
     my ( $gi, $ip_address ) = @_;
     my $seek_org = $gi->_seek_country( addr_to_num($ip_address) );
+
+    return $gi->_common_org($seek_org);
+}
+
+
+sub org_by_addr_v6 {
```

Comment:
Do you think this and the other new `_v6` method should be inside the `__IPV6__` block?